docs/docs/cmd/spo/page/page-control-set.md

```diff
@@ -0,0 +1,55 @@
+# spo page control set
+
+Allows you to set or update control web part data or properties on a modern page.
```

Comment:
In the context of CLI `set` and `update` are the same, so let's use just `update` for simplicity and consistency with other `set` commands